utils.py

```diff
@@ -451,3 +451,9 @@ def calc_hotteling_statistic_for_sample(observed_features: np.ndarray, sample_fe
                         num_features * (sample_size - 1))) * hotelling_t_stat
     return hotelling_t_as_f
 
+@njit
+def generate_binomial_tensor(net_size, num_samples):
```

Comment:
Really not critical, but we can pass `p` (network density) as a parameter as well